solidjs-tailwind/src/components/FileExplorer/FileExplorer.jsx

```diff
@@ -0,0 +1,52 @@
+import { For } from 'solid-js';
+import { Link } from '@solidjs/router';
+import { removeLastPathPart } from '../../utils/removeLastPathPart';
+import { document } from 'solid-heroicons/outline';
+import { folder } from 'solid-heroicons/solid';
+import { Icon } from 'solid-heroicons';
+import styles from './FileExplorer.module.css';
+
+const FileExplorerView = (props) => {
+  return (
+    <div class={styles.container}>
+      {props.repoPath && (
+        <Link
+          href={`${props.basePath}/tree/${props.branch}/${removeLastPathPart(
```

Comment:
lets take this to a variable for readability.

Reply:
I think this can be overlooked, there isn't much happening here for us to consider making it into a variable.